Gemfile

```diff
@@ -1,58 +1,33 @@
 source 'https://rubygems.org'
-git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+git_source(:github) { |repo| "https://github.com/mmrshk/innovation_forpost" }
```

Comment:
Before updating please read what does it mean.
https://stackoverflow.com/questions/41454333/meaning-of-new-block-git-sourcegithub-in-gemfile

It is used to declare such code:
`gem 'foo_bar', :github => 'foo/bar'`

And with your changes it will always try to take from this link - `"https://github.com/mmrshk/innovation_forpost"`